### PR TITLE
Lesson02

### PR DESCRIPTION
**Pull request by maximlip**
Home task 1. Part 1.

### CODE REVIEW

**Comment by a-vodka** [comment]
Please rename comments for all commits in following way.
`Topic: <Action> <the rest title text>`
for example
`Lesson 2 Part 1: Add main() function`

Please, check code style for function braces
https://www.kernel.org/doc/html/v4.10/process/coding-style.html

**Comment by maximlip** [comment]
Home task 1. Part 1.

**Comment by maximlip** [comment]
After rebase -i

**Comment by a-vodka** [comment]
You have only one commit. Please, divide your code into several commits.

**Comment by marynasergiyenko** [comment]
@maximlip , please divide your patch in several comments with appropriate summaries and descriptions.

**Comment by maximlip** [comment]
divided in 3 commits, summaries and descriptions are apdated

File: 01-Git_Bash_Make/dz1.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+int main(void) 
+{
+   
+        if (game()) {
+                printf("You win\n");
+                return 0;
+        }
+        else {
+                printf("You loose\n");
+                return 1;
```
Comment (a-vodka) [inline 01-Git_Bash_Make/dz1.c]: return 1 it a way to say other programs that your program is crashed with error code 1. Please, remove all unnecessary returns. Be careful with coding style.

**Comment by maximlip** [comment]
removed all unnecessary returns

**Pull request merged**